_posts/2024-05-23-wf-32-glow-blog-infoq.adoc

```diff
@@ -0,0 +1,10 @@
+---
+layout: post
+title: 'Blog: WildFly 32 Delivers Preview of Jakarta EE 11 and Final Version of WildFly Glow'
+date: 2024-05-23
+tags: wildfly glow galleon openshift provisioning
+synopsis: 
+author: jfdenise
```

Comment:
Good point, once I have a bio, I will add him as author.

Reply:
mmmh, I interpret `author` as the author of the blog post, not necessarily the author of the things that is linked.

Reply:
I added aurhor content and changed the date to today

Reply:
@jmesnil I think the version that was published is now correct, on wildfly.org it now says:

> By Syed M. Shaaf | May 27, 2024

Otherwise it would have claimed to be by Jean-Francois and when you click the link you find it is actually by a different author.

Reply:
Jean-Francois is the author of the "blog post" and Shaaf is the author of the "thing" the post links to.

With the way things are displayed now, it looks like Shaaf is an author of wildfly.org which is not correct.

This would create a precedent that we would have to create an `author` every time we want to link to another web site which does not seem correct to me.

Reply:
But there is no blog post on wildfly.org just a placeholder in the news stream that links directly to the infoq article.

![image](https://github.com/wildfly/wildfly.org/assets/481922/3eca90ac-a404-42f5-9d8e-1e90646a852a)

For all of the other items on this page the "By" identifies who wrote the content you are about to access.